### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by SindhuGouru**
Please verify

### CODE REVIEW

File: employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class EmployeeController {
+
+    @PostMapping("/employeeDetails")
+//    @ResponseBody
+    public ResponseEntity<String> employeeDetails(@RequestBody Employee employee){
```
Comment (gourusumanth) [inline employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java]: Please change the name of this method to createEmployee as you are creating an employee.
Please update the request mapping path too!

File: employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        System.out.println("Employee Id : "+employee.getEmpId());
```
Comment (gourusumanth) [inline employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java]: please add spacing before and afer +. Thats a good practice!!

File: employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        System.out.println("Employee Name : "+employee.getEmpName());
+        System.out.println("Employee Salary : "+employee.getEmpSalary());
+      return new ResponseEntity<>("Employee created with status 201",HttpStatus.CREATED);
```
Comment (gourusumanth) [inline employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java]: You dont need to send a response for POST endpoint. 

But if you want to you need not mention "Employee created with status 201"  as you are already responding with 201. i would rather give a more meaningful message to the client

File: employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
+package com.artimes.employee.controller;
```
Comment (gourusumanth) [inline employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java:1]: Please add the Junit Tests for this class.

File: employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java (hunk 1 above)
Comment (SindhuGouru) [reply employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java]: Changed method name and request mapping to createEmployee .
Please verify and let me know if any changes required.

File: employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java (hunk 2 above)
Comment (SindhuGouru) [reply employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java]: Thanks for the comment . Added space before and after +

File: employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java (hunk 3 above)
Comment (SindhuGouru) [reply employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java]: Changed comment

File: employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java (hunk 4 above)
Comment (SindhuGouru) [reply employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java:1]: I dont have any functionality to write Junit test cases

**Review by gourusumanth — APPROVED** [review]

**Pull request merged**